redaxo/src/addons/debug/install.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$addon = rex_addon::get('debug');
+rex_dir::copy(
+    $addon->getPath('vendor/itsgoingd/clockwork/Clockwork/Web/public'),
+    $addon->getAssetsPath('clockwork')
+);
+
+$jsFile = $addon->getAssetsPath('clockwork/js/app.fe8ebfde.js');
```

Comment:
Müssen wir hier dann bei Updates immer aufpassen, da sich der hash ändert?

Reply:
jup, aber war auch erstmal quick-and-dirty von mir.

Bin noch am überlegen, das JS selbst zum kompilieren, da das URL ersetzen auch in Zukunft sonst broken sein könnte. Ich schau mal...

Reply:
ich würds erstmal so lassen, da das passende frontend mit dem backend kommt. Da man die Updates eh nur manuell hier rein packt, würde ich einen passenden kommentar in der install.php ergänzen, welche steps zu machen sind beim updaten bzw. was zu beachten ist.